pkg/capture/capture.go

```diff
@@ -255,7 +211,7 @@ type Capture struct {
 	// has Close been called on the Capture?
 	closed bool
 
-	state State
+	state types.State
```

Comment:
Also a bit ambiguous, probably should be `types.CaptureState` (or not be a type in `types` at all - after all it has only to do with `capture` - if you're concerned about any API dependency on slimcap I'd recommend moving it to a type (or whatever) subpackage of `capture` instead).